src/compiler/checker.ts

```diff
@@ -23128,6 +23128,13 @@ namespace ts {
                 !(getModifierFlags(parameter) & ModifierFlags.ParameterPropertyModifier);
         }
 
+        function isOptionalUninitializedParameterProperty(parameter: ParameterDeclaration) {

```

Comment:
Also for consistency; I think it would be really weird for two predicates that are almost opposites not to be next to each other.